src/csv_handler.rs

```diff
@@ -179,4 +188,28 @@ mod tests {
         let result = save_log_to_file::<Data>(invalid_path.as_str(), TEST_COMPONENT);
         assert!(result.is_err());
     }
+
+    macro_rules! test_x_log_to_file {
+        ($($name:ident : ($function:expr,$component_name:expr)),*) => {
+        $(
+            #[test]
+            fn $name() {
+                $function();
+
+                assert!(Path::new(&concat_path($component_name)).exists());
+                cleanup_test_file($component_name);
+            }
+        )*
```

Comment:
I guess you need to update it here as well.
```suggestion
        )+
```

Reply:
Done